### DISCUSSION TITLE
not run so-test Security onion 2.3.30

### DISCUSSION THREAD

**Discussion by padawan00** [Security-Onion-Solutions/securityonion#7983] [Unsupported Versions]
Hello

can help me with my Installation of Security Onion, no i can to run so-test,  generates an error 

$ sudo so-test
Replay functionality not enabled; attempting to enable now (may require Internet access)...
so-tcpreplay image exists.
=========================================================================
Starting tcpreplay...
This could take a while if another Salt job is running.
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================
Replay functionality not enabled;Replay functionality not enabled;^Clocal:
    Data failed to compile:
----------
    Pillar failed to render with the following messages:
----------
    Rendering SLS 'minions.aurora_standalone' failed. Please see master log for details.
ERROR: Unable to initialize tcpreplay
Exiting.

**Comment by dougburks** [comment]
> Rendering SLS 'minions.aurora_standalone' failed. Please see master log for details.

Per the message, have you checked the master log in `/opt/so/log/salt/` for additional clues?
